java/openmldb-spark-connector/src/main/java/com/_4paradigm/openmldb/spark/write/OpenmldbBatchWrite.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com._4paradigm.openmldb.spark.write;
+
+import org.apache.spark.sql.connector.write.BatchWrite;
+import org.apache.spark.sql.connector.write.DataWriterFactory;
+import org.apache.spark.sql.connector.write.LogicalWriteInfo;
+import org.apache.spark.sql.connector.write.PhysicalWriteInfo;
+import org.apache.spark.sql.connector.write.WriterCommitMessage;
+
+public class OpenmldbBatchWrite implements BatchWrite {
+    private final OpenmldbWriteConfig config;
+    private final LogicalWriteInfo info;
+
+    public OpenmldbBatchWrite(OpenmldbWriteConfig config, LogicalWriteInfo info) {
+        this.config = config;
+        this.info = info;
+    }
+
+    @Override
+    public DataWriterFactory createBatchWriterFactory(PhysicalWriteInfo info) {
+        return new OpenmldbDataWriterFactory(config);
+    }
+
+    @Override
+    public void commit(WriterCommitMessage[] messages) {
+
```

Comment:
Should we support batch write? Or throw an unsupported exception.

Reply:
Data Source V2's write only support batch write/streaming write. Online storage can't support batch write(can't do transaction write), but we should use the batch write API of DataSourceV2.

Reply:
It seems that batch write is the basic function in jdbc. We may support batch write in spark once `openmldb-jdbc` supports it.